qa/L0_long_running_stress/stress.py

```diff
@@ -0,0 +1,610 @@
+# Copyright 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import sys
+sys.path.append("../common")
+
+import argparse
+from builtins import range
+from builtins import str
+from future.utils import iteritems
+import os
+import time
+import threading
+import traceback
+import numpy as np
+import test_util as tu
+from functools import partial
+import tritongrpcclient as grpcclient
+from tritonclientutils import np_to_triton_dtype
+
+if sys.version_info >= (3, 0):
+    import queue
+else:
+    import Queue as queue
+
+FLAGS = None
+CORRELATION_ID_BLOCK_SIZE = 100
+DEFAULT_TIMEOUT_MS = 5000
+SEQUENCE_LENGTH_MEAN = 16
+SEQUENCE_LENGTH_STDEV = 8
+
+_thread_exceptions = []
+_thread_exceptions_mutex = threading.Lock()
+
+
```

Comment:
We want the scripts to run for specific backends. We need to capture and condition on the BACKENDS env var here as well. Look at how this in done in L0_infer/infer_test.py

Reply:
Modified, please check the updated PR. Originally the test was only using custom model. Will add other trials if needed.